docs/ABOUT.md

```diff
@@ -38,6 +38,17 @@ Another common implementation is `mawk`, which is based on a bytecode interprete
 This track will use `gawk`, GNU's awk implementation.
 `gawk` may not have the raw speed of `mawk`, but it is very feature-packed, well documented, and has some handy command-line options.
 
+### shebang
+
+Depending on the platform, awk scripts can be executed by adding a "shebang" line to the top of the file.
```

Comment:
For the uninitiated perhaps a link to https://en.wikipedia.org/wiki/Shebang_(Unix)